client/my-sites/people/people-profile/index.jsx

```diff
@@ -1,31 +1,46 @@
 /**
  * External dependencies
  */
-import { connect } from 'react-redux';
-import React, { Component, Fragment } from 'react';
+import React, { Fragment } from 'react';
```

Comment:
```suggestion
import React from 'react';
```

I've left a comment in the conversation tab, where you can check how to remove this Fargment import 🙂